src/gas/GasFeeController.ts

```diff
@@ -115,24 +114,36 @@ export interface GasFeeEstimates {
   estimatedBaseFee: string;
 }
 
-function isEIP1559Estimate(object: any): object is GasFeeEstimates {
-  return (
-    'low' in object &&
-    isEIP1559GasFee(object.low) &&
-    'medium' in object &&
-    isEIP1559GasFee(object.medium) &&
-    'high' in object &&
-    isEIP1559GasFee(object.high) &&
-    'estimatedBaseFee' in object
-  );
-}
-
 const metadata = {
   gasFeeEstimates: { persist: true, anonymous: false },
   estimatedGasFeeTimeBounds: { persist: true, anonymous: false },
   gasEstimateType: { persist: true, anonymous: false },
 };
 
```

Comment:
So, just to confirm I understand: state will always have a `estimatedGasFeeTimeBounds` property, but it will be an empty object unless `isEIP1559Compatible` is true?

Reply:
Yep, all fields are always present, but their shape is determined by the value of gasEstimateType.

Reply:
and the fee-market value will only be set if we get a successful response from the EIP1559 gas estimate API.